### ISSUE TITLE
A11y_Visual studio Code Client_Editor_More actions_Non Text Contrast: Focus is not clearly visible on "more actions" and items present inside "More actions" button ratio is 1.48:1 less than required

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#193198]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
 #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2023; #WCAG1.1.1; #Non Text Content; 

**Environment and OS details:**
Visual Studio Code 
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992

**Repro Steps:**
Open Visual Studio Code 
Tab till More button present in editor section.
Select More actions button. 
Verify whether Focus is not clearly visible on "more actions" and items present inside "More actions" button or not.

**Actual Result:**
Focus is not clearly visible on "more actions" and items present inside "More actions" for example "Show open Editors", "Close All", "Close saved" etc. buttons ratio is 1.48:1 less than required 3:1.

**Expected Result:**
Focus should be clearly visible on "more actions" and items present inside "More actions" for example "Show open Editors", "Close All", "Close saved" etc. buttons ratio should be 3:1 or greater than 3:1.

**Attachment:**
<img width="942" alt="External to VS Code" src="https://github.com/microsoft/vscode/assets/98530069/40ef90cb-9f5a-4628-96b9-dfecad0bac61">

**Comment by meganrogge** [comment]
@daviddossett would appreciate your thoughts on this 🙏🏼

**Comment by meganrogge** [comment]
also cc @isidorn

**Comment by daviddossett** [comment]
Hm, I see the usual focus border on focus and after hitting enter:

<img width="71" alt="CleanShot 2023-09-18 at 09 00 57@2x" src="https://github.com/microsoft/vscode/assets/25163139/02641e5f-a6c6-42ca-addd-33358b396670">

<img width="319" alt="CleanShot 2023-09-18 at 09 01 00@2x" src="https://github.com/microsoft/vscode/assets/25163139/473d63a7-d833-42d7-ab4f-cf64c646d3d0">

On the second issue re: menu items not meeting contrast ratio minimums—is the issue here that the background color itself needs to have greater contrast? My guess is that we have to play a balancing act with the text contrast in that case. Fwiw this is the same color that Dark+ has been using forever. In fact the menu backgrounds are darker in Dark Modern so it should be even higher contrast here.

**Comment by meganrogge** [comment]
Yeah, I am not sure that this is an issue. We have our high contrast themes that can be used if this doesn't meet someone's needs.

**Comment by daviddossett** [comment]
I know there are some guidelines around non-text elements meeting a 3:1 bar but I don't recall the specific for this exact case. I know there are some exceptions for things like borders but perhaps list item states are held to the 3:1 bar.

**Comment by meganrogge** [comment]
These are met when using our high contrast themes, so I think we can close this.

**Comment by kupatkar99** [comment]
@meganrogge when we are verifying we are not getting the dotted border we are getting the blue focus while navigating for which the color contrast ratio is less.

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Visual Studio Code Version: 1.106.3 (User Setup)
Issue is fixed. hence Closing the bug. Please find below added attachment for reference.

<img width="943" height="320" alt="Image" src="https://github.com/user-attachments/assets/8d04d726-7c98-4b7b-ad53-6c217753c6e3" />
#Closed;
